# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/moz.build
# Commit: 2fc9a3d0ad8b
# Full Hash: 2fc9a3d0ad8bdbce325df78fcd731e925245afaf
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-12 09:23:07
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 1 - CookieStore API - WebIDL, r=smaug,edgul
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D215144
# ==============================================================================

diff -r 2d598be5ec67 -r 2fc9a3d0ad8b dom/moz.build
--- a/dom/moz.build	Wed Sep 11 16:09:37 2024 +0000
+++ b/dom/moz.build	Wed Sep 11 16:26:54 2024 +0000
@@ -37,6 +37,7 @@
     "chrome-webidl",
     "clients",
     "commandhandler",
+    "cookiestore",
     "credentialmanagement",
     "crypto",
     "debugger",